hack/make.sh

```diff
@@ -333,6 +333,19 @@ copy_containerd() {
 	fi
 }
 
+install_binary() {
+	file="$1"
+	target="${DOCKER_MAKE_INSTALL_PREFIX:=/usr/local}/bin/"
+	if [ "$(go env GOOS)" == "linux" ]; then
+		echo "Installing $(basename $file) to ${target}"
+		cp $file $target
```

Comment:
These arguments ought to be quoted (ie, `cp "$file" "$target"`), and adding `-L` is probably also prudent to ensure we copy/install files, not symlinks (`cp -L "$file" "$target"`).  Sorry I didn't notice this earlier! :tired_face: